Rakefile

```diff
@@ -3,6 +3,13 @@ require 'suse/connect'
 require 'rspec/core/rake_task'
 require 'date'
 
+module MonkeypatchRakeLastComment
```

Comment:
I would miss a hint on the reasons why that monkey patching is needed. Could you provide one?